index.html

```diff
@@ -771,6 +771,20 @@ <h3>
           Details of how to actually get the [=digital credential=] are
           forthcoming.
         </aside>
+        <ol class="algorithm" >
+          <li>
+            <aside class="issue">
+              The following guidance on handling multiple requests will need to be relocated once
+              this entire section of the algorithm is defined.
+            </aside>
+            When multiple [=digital credential/presentation requests=] are present
+            in {{DigitalCredentialRequestOptions/requests}},
+            [=user agents=] MUST NOT return more than one
+            [=digital credential/presentation responses | presentation response=].
```

Comment:
@TallTed Here's the latest version of the text we were [discussing in a previous comment chain](https://github.com/w3c-fedid/digital-credentials/pull/249#discussion_r2195765174).

Reply:
Response to full sentence is in https://github.com/w3c-fedid/digital-credentials/pull/249/files#r2308148399